packages/council-graphql/src/datasources/LockingVaultContract.ts

```diff
@@ -14,4 +17,53 @@ export class LockingVaultContract extends VotingVaultContract {
     super(address, contract);
     this.contract = contract;
   }
+
+  async getVotingPowerView(
```

Comment:
Let's document this one as well:

```ts
/**
Get's the voter's voting power at a given block without taking into account the staleBlockLag. This is useful for showing the user's current voting power in the system.

If you wish to see the user's voting power for a given proposal, use getVotingPower instead.
*/
```

Reply:
Added to the interface so it will be visible by all consumers of voting vault data sources.
![image](https://user-images.githubusercontent.com/3289505/178645887-21192df2-5b54-406e-8cfa-3e266b223b42.png)